src/db.ts

```diff
@@ -0,0 +1,54 @@
+import mongoose, { Schema, Document, Model } from 'mongoose';
+import { mongodb } from './config.json';
+import logger from './utils/logger'
+
+
+
+const uri: string = mongodb.url;
+
+const connectDB = async (): Promise<void> => {
+    try {
+        await mongoose.connect(uri);
+        logger.info('MongoDB connected successfully');
+    } catch (error) {
+        logger.error('MongoDB connection error:', error);
+    }
+};
+
+// Define the interface for the document
+interface IUserData extends Document {
+    discordUserId: string;
+    refreshToken: string;
+}
+
+const userDataSchema: Schema<IUserData> = new Schema({
+    discordUserId: {
+        type: String,
+        required: true,
+        unique: true,
+    },
+    refreshToken: {
+        type: String,
+        required: true,
+    }
+});
+
+const userDataModel: Model<IUserData> = mongoose.model<IUserData>('userRefreshToken', userDataSchema);
```

Comment:
The collection name `userRefreshToken` doesn't make sense. Conventionally, it's the same as the name of the model, so it could be `UserData` or `User`.

Reply:
I will take a look at that.